src/Algebra/Construct/LexProduct.agda

```diff
@@ -107,6 +107,4 @@ sel ∙-sel ◦-sel (a , x) (b , y) with (a ∙ b) ≟₁ a | (a ∙ b) ≟₁ b
 ... | no  ab≉a | no  ab≉b  = contradiction₂ (∙-sel a b) ab≉a ab≉b
 ... | yes ab≈a | no  _     = inj₁ (ab≈a , ≈₂-refl)
 ... | no  _    | yes ab≈b  = inj₂ (ab≈b , ≈₂-refl)
-... | yes ab≈a | yes ab≈b  with ◦-sel x y
-...   | inj₁ xy≈x = inj₁ (ab≈a , xy≈x)
-...   | inj₂ xy≈y = inj₂ (ab≈b , xy≈y)
+... | yes ab≈a | yes ab≈b  = map (ab≈a ,_) (ab≈b ,_) $ ◦-sel x y
```

Comment:
Potato, aardappel...

I'd be content with `(◦-sel x y)` as the final argument, but I suppose this idiomatic usage displays that term (slightly?) more prominently. 
happy to leave as-is, but I tend to think such usage shines more when deployed with a multi-line equational proof as the trailing argument, where parentheses *do* clutter things up. 

No big deal here.

Reply:
Done